java/spring-framework/configure-spring-boot-starter-java-app-with-azure-active-directory-b2c-oidc.md

```diff
@@ -214,15 +214,15 @@ respectively. Specify your user flow **Name** and **User attributes and claims**
 13. Enter the following code, then save and close the file:
 
     ```java
-    package sample.aad.security;
+    package com.example.demo.security;
     
     import com.microsoft.azure.spring.autoconfigure.b2c.AADB2COidcLoginConfigurer;
     import org.springframework.security.config.annotation.web.builders.HttpSecurity;
     import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
     import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
     
     @EnableWebSecurity
-    public class WebSecurityConfiguration extends WebSecurityConfigurerAdapter {
+    public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
The constructor name would need to be updated also, but I recommend spelling out "WebSecurityConfiguration" for the class name, and updating the file name on line 212 to match. Thanks!

```suggestion
    public class WebSecurityConfiguration extends WebSecurityConfigurerAdapter {
```

Reply:
I went ahead and made the changes directly. Thanks for the pull request!

Reply:
Sorry for my late response and thanks for your helping!